File: Poo/Poo/Poo aulas/Aula 2/Carro.java
public class Carro {
    // atributos
    private String marca;
    private String modelo;
    private int ano;

    // Método construtor
    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    // Método para obter o modelo do carro
    public String getMarca() {
        return marca;
    }

    // Método para obter modelo do carro
    public String getModelo() {
        return modelo;
    }

    // Método para obter o ano do carro
    public int getAno() {
        return ano;
    }

    // definir a marca
    public void setMarca(String marca) {
        this.marca = marca;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public void setAno(int ano) {
        this.ano = ano;
    }
}

File: Poo/Poo/Poo aulas/Aula 2/Carro.py
class Carro:
    def __init__(self, marca, ano, cor, modelo, placa, marcha):
        self.marca = marca
        self.ano = ano
        self.cor = cor
        self.modelo = modelo
        self.placa = placa
        self.marcha = marcha
        self.roda = 4
        self.is_running = False
        self.velocidade = float(0)
    
    def __str__(self):
        return f"O carro é {self.marca}, O ano é {self.ano}"
    
    def start(self):
        if not self.is_running:
            self.is_running = True
            print("O carro esta ligado!!!")
    
    def stop(self):
        if self.is_running:
            self.is_running = False
            print("O carro está desligado!!!")
        
    def drive(self, v2):
        if self.is_running:
            self.velocidade += v2
            print(f"Dirigindo {self.velocidade} km/h")
        else:
            print("O carro precisa ser ligado para acelarar e dirigir!!!")

    
    

